public/css/standard.css

```diff
@@ -0,0 +1,3 @@
+.logo {
+    width: 40%;
+}
```

Comment:
You can set PhpStorm (most editors btw) to auto add a newline at the end of all files. You can look into the reason why, but it comes down to it being a widely accepted standard.